docs/slack.rst

```diff
@@ -259,22 +259,48 @@ In addition to the guidance that applies to all participants, we expect self-pro
   Clearly explain what you are promoting, why it matters to documentarians, and your involvement.
   Be up front about paywalls, registration requirements, referral benefits, or commissions.
   If you're running a survey, say when and how the results will be published.
+
```

Comment:
These spaces don't appear elsewhere. Do you want to add them to _all_ the bulleted lists, or none of them? (My kingdom for a Prettier-like formatter for reStructuredText.)